src/main/webapp/app/exercises/programming/shared/code-editor/file-browser/code-editor-file-browser-badge.component.html

```diff
@@ -0,0 +1,4 @@
+<span class="badge" [class.on-colored-background]="onColoredBackground" [ngbTooltip]="tooltip">
```

Comment:
Note for code reviewers: That's the badge on the right:
![screenshot-2023-08-28_001851](https://github.com/ls1intum/Artemis/assets/9006596/caec0d2b-9bcf-4eab-a4bb-dfa12360a364)